apps/aecore/lib/aecore/sync/chain.ex

```diff
@@ -131,18 +131,18 @@ defmodule Aecore.Sync.Chain do
     |> Enum.reverse()
   end
 
-  defp merge([], [elem2 | chain_2], acc) do
-    merge([], chain_2, [elem2 | acc])
+  defp merge([], [chain2 | chain_list_2], acc) do
```

Comment:
maybe I don't understand what this function does, could you explain what a chain2 element is?

Reply:
Chain1 and Chain2 are both the same type of element, with this function we just merge both chains to one chain. We get all the different elements from chain1 and chain2 and add the elements to the acc.

Reply:
That does not answer what an element is